View/Utilities/BoolToUnidentifiableConverter.cs

```diff
@@ -8,7 +8,7 @@ public class BoolToUnidentifiableConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            if ((bool)value == true)
+            if (System.Convert.ToBoolean(value) == true)
```

Comment:
Namespacing, you don't need System.

Reply:
You need "System" because the method name is Convert and it gets confused.